src/testing/mocks.rs

```diff
@@ -37,6 +39,12 @@ pub fn mock_transaction(payload: Vec<u8>) -> MockTransaction {
     TestTransaction(payload)
 }
 
+impl QueryableHeader for MockHeader {
+    fn timestamp(&self) -> u64 {
+        random::<u64>() % (Utc::now().timestamp() as u64)
```

Comment:
Why `random`?

Reply:
changed to Utc::now() 
 was trying to insert mock data with range of timestamps into db for testing